### PR TITLE
vmanomaly guide docker compose run fix according licensing

### PR DESCRIPTION
**Pull request by Cambaza**
- add license into docker-compose file
- added gcloud auth instruction
- added where to get license code

### CODE REVIEW

File: docs/guides/guide-vmanomaly-vmalert.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ aliases:
 # Getting started with vmanomaly
 
 **Prerequisites**
+- *vmanomaly* is a part of enterprise package. You can get license key [here](https://victoriametrics.com/products/enterprise/trial) to try this tutorial.
 - In the tutorial, we'll be using the following VictoriaMetrics components:
   -  [VictoriaMetrics](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html) (v.1.83.1)
```
Comment (fred-navruzov) [inline docs/guides/guide-vmanomaly-vmalert.md:16]: should we refresh our own components' versions?
i.e. to 1.93.1, which is [the freshest one according to our releases](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) (but omitting `latest` tag) 

here and throughout this guide

File: docs/guides/guide-vmanomaly-vmalert.md [hunk 2]
```diff
@@ -347,7 +359,10 @@ services:
     restart: always
     volumes:
       - ./vmanomaly_config.yml:/config.yaml
-    command: [ "/config.yaml" ]
+    platform: "linux/amd64"
+    command: 
+      - "/config.yaml"
+      - "--license=YOUR_LICENCE_CODE_IS_HERE"
```
Comment (fred-navruzov) [inline docs/guides/guide-vmanomaly-vmalert.md:365]: after this docker-compose `.yml`, we can add reference to our licensing options, i.e.
'see other ways of how to use license for vmanomaly [here](https://docs.victoriametrics.com/vmanomaly.html#licensing)'

**Review by fred-navruzov — APPROVED** [review]
LGTM with comments

File: docs/guides/guide-vmanomaly-vmalert.md [hunk 3]
```diff
@@ -337,7 +349,7 @@ services:
     restart: always
   vmanomaly:
     container_name: vmanomaly
-    image: us-docker.pkg.dev/victoriametrics-test/public/vmanomaly-trial:v1.3.0
+    image: us-docker.pkg.dev/victoriametrics-test/vmanomaly/vmanomaly:latest
```
[the remark this answers is not in the document]
Comment (fred-navruzov) [reply docs/guides/guide-vmanomaly-vmalert.md:352]: then we can use 
`image: us-docker.pkg.dev/victoriametrics-test/public/vmanomaly-trial:v1.5.0` as `latest` tag will be updated after our customers who pulls ":latest" and do not have license yet, are notified and updated

**Review by tenmozes — APPROVED** [review]

**Pull request merged**